ee/codegen/src/__test__/nodes/__snapshots__/note-node.test.ts.snap

```diff
@@ -11,12 +11,6 @@ from ...nodes.note_node import NoteNode
 
 class NoteNodeDisplay(BaseNoteNodeDisplay[NoteNode]):
     node_id = UUID("<note-node-id>")
-    text = "This is a note"
-    style = {
```

Comment:
DevinAI remove these attributes from `BaseNoteNodeDisplay` and move them to `BaseNoteNode`. Actually just mark them as deprecated and to be removed in v2.0.0 in the former